proxy/http/HttpBodyFactory.cc

```diff
@@ -978,9 +918,9 @@ HttpBodySet::init(char *set, char *dir)
   this->set_name = ats_strdup(set);
 
   if (this->table_of_pages) {
-    delete (this->table_of_pages);
+    this->table_of_pages.reset(nullptr);
```

Comment:
Why do this? Just `reset(new TemplateTable);`.

Reply:
I guess the logic was there because it's possible that `this->table_of_pages` is already set by something else. Just guessing, I'll get rid of it.

Reply:
Yes, that would have mattered without `std::unique_ptr`.